lib/services/authentication.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:convert';
+import 'dart:async';
+import 'package:http/http.dart' as http;
+
+class Authentication {
+  final String token;
+  final bool valid;
+
+  Authentication({this.token, this.valid});
+
+  factory Authentication.fromJson(Map<String, dynamic> json) {
+    return Authentication(token: json['jwt'], valid: true);
+  }
+}
+
+Future<Authentication> fetchAuthToken(String email, String password) async {
+  
+  final response = await http.post('https://safira20.herokuapp.com/api/auth/sign_in?email=' + email + '&password=' + password);
```

Comment:
The api url should be on .env file

Reply:
I thought about that but didn't comment since it isn't the aim of this PR. I do agree it could be done here.
Up to you guys.

Reply:
The api url should be in a env file or even on a Service object. 
It should be a hardcoded or even in multiple points in the code because a simples domain change and you need to do a hole commit changing it. It should only depend on the env in what the app is build on.
The route should stay though.
Something like

`url[ENV] + '/api/auth/sign_in?email=' + email + '&password=' + password`

Reply:
My point wasn't that it isn't needed. Only that he might prefer to do it in the next PR/a PR just for it. 

Why should the route stay?
I'd put `/api/auth/sign_in` in the env file as well.

Reply:
I agree with you @apedrob, it makes the app better structured.I will create one and update this PR.

Reply:
I think the route should stay because its not duplicated anywhere else, and its not interchangeable by env. You are defining a middleware for the service, domains change, apis dont unless its a breaking change and that should be changed all around

Reply:
My train of thought was since it is API specific-ish (the sign in is pretty generic though) it should go the the env file.

Reply:
@herulume @apedrob @ZePedroResende  what do you think of this approach to environments I've just commited?